tests/test.conf

```diff
@@ -1,2295 +1,2814 @@
 -k ./certs/ecc-key.pem
+-2
 
 # client TLSv1.2 ECDH-RSA-AES128-GCM-SHA256
 -v 3
 -l ECDH-RSA-AES128-GCM-SHA256
+-2
 
 # server TLSv1.2 ECDH-RSA-AES256-GCM-SHA384
 -v 3
 -l ECDH-RSA-AES256-GCM-SHA384
 -c ./certs/server-ecc-rsa.pem
 -k ./certs/ecc-key.pem
+-2
 
 # client TLSv1.2 ECDH-RSA-AES256-GCM-SHA384
 -v 3
 -l ECDH-RSA-AES256-GCM-SHA384
+-2
 
 # server TLSv1.2 DHE-RSA-AES128-GCM-SHA256
 -v 3
 -l DHE-RSA-AES128-GCM-SHA256
+-2
 
 # client TLSv1.2 DHE-RSA-AES128-GCM-SHA256
 -v 3
 -l DHE-RSA-AES128-GCM-SHA256
+-2
 
 # server TLSv1.2 DHE-RSA-AES256-GCM-SHA384
 -v 3
 -l DHE-RSA-AES256-GCM-SHA384
+-2
 
 # client TLSv1.2 DHE-RSA-AES256-GCM-SHA384
 -v 3
 -l DHE-RSA-AES256-GCM-SHA384
+-2
 
 # server TLSv1.2 PSK-AES128-GCM-SHA256
 -s
 -v 3
 -l PSK-AES128-GCM-SHA256
+-2
 
 # client TLSv1.2 PSK-AES128-GCM-SHA256
 -s
 -v 3
 -l PSK-AES128-GCM-SHA256
+-2
 
 # server TLSv1.2 PSK-AES256-GCM-SHA384
 -s
 -v 3
 -l PSK-AES256-GCM-SHA384
+-2
 
 # client TLSv1.2 PSK-AES256-GCM-SHA384
 -s
 -v 3
 -l PSK-AES256-GCM-SHA384
+-2
 
 # server TLSv1.2 AES128-CCM-8
 -v 3
 -l AES128-CCM-8
+-2
 
 # client TLSv1.2 AES128-CCM-8
 -v 3
 -l AES128-CCM-8
+-2
 
 # server TLSv1.2 AES256-CCM-8
 -v 3
 -l AES256-CCM-8
+-2
 
 # client TLSv1.2 AES256-CCM-8
 -v 3
 -l AES256-CCM-8
+-2
 
 # server TLSv1.2 ECDHE-ECDSA-AES128-CCM
 -v 3
 -l ECDHE-ECDSA-AES128-CCM
 -c ./certs/server-ecc.pem
 -k ./certs/ecc-key.pem
+-2
 
 # client TLSv1.2 ECDHE-ECDSA-AES128-CCM
 -v 3
 -l ECDHE-ECDSA-AES128-CCM
 -A ./certs/ca-ecc-cert.pem
+-2
 
 # server TLSv1.2 ECDHE-ECDSA-AES128-CCM-8
 -v 3
 -l ECDHE-ECDSA-AES128-CCM-8
 -c ./certs/server-ecc.pem
 -k ./certs/ecc-key.pem
+-2
 
 # client TLSv1.2 ECDHE-ECDSA-AES128-CCM-8
 -v 3
 -l ECDHE-ECDSA-AES128-CCM-8
 -A ./certs/ca-ecc-cert.pem
+-2
 
 # server TLSv1.2 ECDHE-ECDSA-AES256-CCM-8
 -v 3
 -l ECDHE-ECDSA-AES256-CCM-8
 -c ./certs/server-ecc.pem
 -k ./certs/ecc-key.pem
+-2
 
 # client TLSv1.2 ECDHE-ECDSA-AES256-CCM-8
 -v 3
 -l ECDHE-ECDSA-AES256-CCM-8
 -A ./certs/ca-ecc-cert.pem
+-2
 
 # server TLSv1.2 PSK-AES128-CCM
 -s
 -v 3
 -l PSK-AES128-CCM
+-2
 
 # client TLSv1.2 PSK-AES128-CCM
 -s
 -v 3
 -l PSK-AES128-CCM
+-2
 
 # server TLSv1.2 PSK-AES256-CCM
 -s
 -v 3
 -l PSK-AES256-CCM
+-2
 
 # client TLSv1.2 PSK-AES256-CCM
 -s
 -v 3
 -l PSK-AES256-CCM
+-2
 
 # server TLSv1.2 PSK-AES128-CCM-8
 -s
 -v 3
 -l PSK-AES128-CCM-8
+-2
 
 # client TLSv1.2 PSK-AES128-CCM-8
 -s
 -v 3
 -l PSK-AES128-CCM-8
+-2
 
 # server TLSv1.2 PSK-AES256-CCM-8
 -s
 -v 3
 -l PSK-AES256-CCM-8
+-2
 
 # client TLSv1.2 PSK-AES256-CCM-8
 -s
 -v 3
 -l PSK-AES256-CCM-8
+-2
 
 # server TLSv1.0 DHE-PSK-AES128-CBC-SHA256
 -s
 -v 1
 -l DHE-PSK-AES128-CBC-SHA256
+-2
 
 # client TLSv1.0 DHE-PSK-AES128-CBC-SHA256
 -s
 -v 1
 -l DHE-PSK-AES128-CBC-SHA256
+-2
 
 # server TLSv1.1 DHE-PSK-AES128-CBC-SHA256
 -s
 -v 2
 -l DHE-PSK-AES128-CBC-SHA256
+-2
 
 # client TLSv1.1 DHE-PSK-AES128-CBC-SHA256
 -s
 -v 2
 -l DHE-PSK-AES128-CBC-SHA256
+-2
+
+# server TLSv1.2 DHE-PSK-AES128-CBC-SHA256 (DHE prime test)
```

Comment:
@toddouska : The only DHE Prime Test is here. @ejohnstown : Should we test some other cipher suites as well?

Reply:
There are three cipher suites with DHE testing in test.conf: DHE-RSA-AES128-SHA, DHE-RSA-AES256-SHA256, and DHE-PSK-AES128-CBC-SHA256.

Reply:
ok, thanks